### PR TITLE
adjusted string representations and add a few variables for speed

### PR DESCRIPTION
**Pull request by simurgh9**

### CODE REVIEW

**Comment by amymcgovern** [comment]
Please turn noise back on

File: aiclass/droneSimulator.py [hunk 1]
```diff
@@ -13,16 +13,19 @@
...
+max_asteroid_speed = 0.5  # m/s
+default_move_speed = 0.5  # m/s
 asteroid_radius = 0.1  # in meters (this is large initially)
 drone_radius = 0.05  # in meters
 score_timesteps = 100  # number of time ticks between scores on the same pad
 crash_timesteps = 30 # number of time tickets between crashes on the same pad
 asteroid_damage = 5 # constant damage score for hitting a non-mineable asteroid
-move_noise = 0.1 # noise variance in x, y, z in m/s
-turn_noise = 0.02 # noise variance in radians
+move_noise = 0  # 0.1 # noise variance in x, y, z in m/s
```
Comment (amymcgovern) [inline aiclass/droneSimulator.py]: Please leave the noise on for the check in

**Comment by simurgh9** [comment]
I did.

**Comment by amymcgovern** [comment]
thanks.

**Pull request merged**